oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticDocumentMaker.java

```diff
@@ -57,6 +58,7 @@ protected ElasticDocument finalizeDoc(ElasticDocument doc, boolean dirty, boolea
         // Evaluate path restrictions is enabled by default in elastic. Always index ancestors.
         // When specifically disabled, we will keep indexing it, but the field won't be used at query time
         doc.indexAncestors(path);
+        doc.setLastUpdated(Instant.now().toEpochMilli());
```

Comment:
As this method is an hot spot, we could use directly System.currentTimeMillis(), which should be much faster than creating an Instant and converting it to millis.